ui/src/common/utils.js

```diff
@@ -208,27 +208,9 @@ export function downloadTextAsFile(
   type = 'application/txt',
   extension = 'txt'
 ) {
+  const date = moment().format('YYYY-MM-DDTHHmmss');
   const blob = new Blob([text], { type });
-
-  if (window.navigator && window.navigator.msSaveOrOpenBlob) {
-    // Edge
-    window.navigator.msSaveOrOpenBlob(blob, `download.${extension}`);
-  } else if (browser.isSafari()) {
-    // Safari
-    const url = URL.createObjectURL(blob);
-    const a = document.createElement('a');
-    a.style.display = 'none';
-    a.href = url;
-    a.download = `download.${extension}`;
-    document.body.appendChild(a);
-    a.click();
-    window.URL.revokeObjectURL(url);
-  } else {
-    // the rest
-    const url = URL.createObjectURL(blob);
-    window.open(url, '_self');
-    URL.revokeObjectURL(url);
-  }
+  saveAs(blob, `INSPIRE-Cite_${date}.${extension}`);
```

Comment:
this is a generic function used to save anything, `INSPIRE-Cite` doesn't make sense. 

`downloadTextAsFile` can just take file name (together with the extension)

Reply:
this is still not addressed.